### DISCUSSION TITLE
Parent scope cannot be accessed from within the init() function of Alpine.data

### DISCUSSION THREAD

**Discussion by pomartel** [alpinejs/alpine#1711] [7. Bugs (archived)]
As demonstrated in this [codepen](https://codepen.io/pomartel/pen/GRmJvVz), the parent scope is not accessible from within the `init()` function of `Alpine.data()`.

However the parent scope is present when the `x-init` directive is used as demonstrated [here](https://codepen.io/pomartel/pen/yLbYBzg).

*Expected behaviour:* The `init()` function should behave like `x-init` and have access to the parent scope.

*Actual behaviour:* The `init()` function does not have access to the parent scope.

**Comment by alexej-d** [comment]
Duplicate of [this](https://github.com/alpinejs/alpine/discussions/1674)
